bin/propolis-server/src/lib/spec/mod.rs

```diff
@@ -186,6 +226,16 @@ pub struct SoftNpu {
     pub p9fs: Option<P9fs>,
 }
 
+#[cfg(feature = "falcon")]
+impl SoftNpu {
+    pub fn is_empty(&self) -> bool {
```

Comment:
considered pedantic, feel free to ignore: dunno if i love `is_empty()`, which kind of treats this as a generic collection; perhaps something more semantically meaningful like `is_softnpu_enabled()` or similar? :woman_shrugging: